models/public/vehicle-reid-0001/vehicle-reid-0001.md

```diff
@@ -0,0 +1,76 @@
+# vehicle-reid-0001
+
+## Use Case and High-Level Description
+
+This is a vehicle reidentification model for a general scenario. It uses a whole
+car body image as an input and outputs an embedding vector to match a pair of images
+by the cosine distance. The model is based on the OmniScaleNet backbone developed for fast inference.
+A single reidentification head from the 1/16 scale
+feature map outputs an embedding vector of 256 floats.
+
+## Specification
+
+| Metric                            | Value                                     |
+|-----------------------------------|-------------------------------------------|
+| VeRi-776\* rank-1                 | 96.31 %                                   |
+| VeRi-776\* mAP                    | 85.15 %                                   |
+| Camera location                   | All traffic cameras                       |
+| Support of occluded vehicles      | YES                                       |
+| Occlusion coverage                | <50%                                      |
+| GFlops                            | 2.643                                     |
+| MParams                           | 2.183                                     |
+| Source framework                  | PyTorch\*                                 |
+
+The cumulative matching curve (CMC) at rank-1 is accuracy denoting the possibility
+to locate at least one true positive in the top-1 rank.
+Mean Average Precision (mAP) is the mean across Average Precision (AP) of all queries.
+AP is defined as the area under the
+[precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
+
+## Performance
+
+## Inputs
```

Comment:
Since model is downloaded not in IR format, you should describe I\O both for original (ONNX) and converted models.

Reply:
Fixed